dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -110,6 +112,15 @@ public static LinkageChecker create(Bom bom, Path exclusionFile)
         classPathBuilder.resolve(managedDependencies, true, DependencyMediation.MAVEN);
     ImmutableList<ClassPathEntry> classpath = classPathResult.getClassPath();
 
+    ImmutableList<UnresolvableArtifactProblem> artifactProblems = classPathResult
+        .getArtifactProblems();
+    if (!artifactProblems.isEmpty()) {
+      for (UnresolvableArtifactProblem artifactProblem : artifactProblems) {
+        logger.warning(artifactProblem.toString());
+      }
+      throw new IOException("Could not resolve dependencies");
```

Comment:
This might break project with complex dependency tree such as Beam and spring-cloud-gcp.

Reply:
Confirmed that these are not the case. Beam's usage (LinkageCheckerMain with "-a" option) does not use this code path. Spring-cloud-gcp's usage is through the enforcer rule, which uses ClassPathBuilder before calling LinkageChecker.

This constructor is used by Linkage Monitor and MaximumLinkageErrorsTest.